tests/sentry_plugins/amazon_sqs/test_plugin.py

```diff
@@ -134,6 +134,9 @@ def test_use_s3_bucket(self, mock_client):
 
     @patch("sentry_plugins.amazon_sqs.plugin.logger")
     @patch("boto3.client")
+    @pytest.mark.skip(
+        reason="Flakey: https://sentry.sentry.io/issues/3901571639/?project=2423079 and https://sentry.sentry.io/issues/3901571639/?project=2423079"
+    )
```

Comment:
let's make sure to create a ticket so it gets fixed at some point -- I want to say ecosystem owns this?

Reply:
Done - added issue to description. I tried to find an owner and I came up blank.

Reply:
ideally we'd have the issue number in the skipped message -- ala https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md